packages/sdk/src/resolve/resolve-accounts.js

```diff
@@ -1,19 +1,19 @@
 import {invariant} from "@onflow/util-invariant"
-import {isTransaction} from "../interaction/interaction.js"
+import {isTransaction, makeVoucher} from "../interaction/interaction.js"
 
 const isFn = v => typeof v === "function"
-
-function buildPreSignable(acct, ix) {
+export function buildPreSignable(acct, ix) {
   try {
     return {
       f_type: "PreSignable",
-      f_vsn: "1.0.0",
+      f_vsn: "1.0.1",
       roles: acct.role,
       cadence: ix.message.cadence,
       args: ix.message.arguments.map(d => ix.arguments[d].asArgument),
       data: {},
       metadata: ix.metadata,
       interaction: ix,
+      voucher: makeVoucher(ix),
```

Comment:
lol, why voucher

Reply:
haha was hitting that thesaurus, thinking of it as cred to be used to verify the message. 
**def:** written authorization; credential.

Reply:
**alt key names**

- proof
- verifier
- validator
- document
- confirm
- messageConf

**alt function names**

- make[Proof][Validator][Verifier]
- prep[Proof][Validator][Verifier]